client/src/components/Questions.tsx

```diff
@@ -0,0 +1,184 @@
+import React from "react";
+import {ThemeProvider, theme, CSSReset, Flex, Box, Image, Heading, HStack, Stack, Button} from "@chakra-ui/react";
+import ReactDOM from 'react-dom';
+export const Questions = () => {
+    return (
+        <ThemeProvider theme={theme}>
+            <CSSReset />
+            <QuestionsPage />
+        </ThemeProvider>
+    );
+}
+
+const QuestionBank = [
+    {
+        questionTitle: "This is a test question?", 
+        A: "Answer A",
+        B: "Answer B",
+        C: "Answer C",
+        D: "Answer D",
+        Correct: "A"
+    },
+    { 
+      questionTitle: "why does this happen?", 
+      A: "ABCD",
+      B: "EFGH",
+      C: "IKLM",
+      D: "NOP",
+      Correct: "C"
+    }
+]
+
+const AnswerStyle = {
+    borderWidth: "3px", 
+    borderColor:  "#000000",
+    borderRadius: "0px",
+    backgroundColor: "orange",
+    color: "black",
+    height: '100px',
+    width: '190px',
+}
+const id = 1
```

Comment:
Can you rename this to questionId? Thanks

Reply:
I have resolved the comments.